tests/locale/keymap_or_locale.pm

```diff
@@ -0,0 +1,97 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Keyboard layout test in console and gdm after boot
+# Maintainer: Martin Loviska <mloviska@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use utils;
+
+sub verify_default_keymap_textmode {
+    my ($test_string, $tag, %tty) = @_;
+    select_console($tty{console}) if defined($tty{console});
+    type_string($test_string);
+    assert_screen($tag);
+    # clear line in order to add user bernhard to tty group
+    # clear line to avoid possible failures in following console tests if scheduled
+    send_key("ctrl-w");
+}
+
+sub list_locale_settings {
+    my $program = shift;
+
+    x11_start_program($program) if (get_var('DESKTOP') && (!check_var('DESKTOP', 'textmode')));
+    script_run('localectl status');
+    script_run('locale');
```

Comment:
Both of these commands should not fail so I would just call `assert_script_run`. By default I recommend the following rule: "If you don't care about the exit status call `assert_script_run`, otherwise check the return code of `script_run` explicitly".